custom/icds_reports/migrations/0014_add_datasource_views.py

```diff
@@ -14,5 +14,4 @@ class Migration(migrations.Migration):
     ]
 
     operations = [
-        migrator.get_migration('create_datasource_views.sql'),
```

Comment:
```create_datasource_views.sql``` references ```awc_location.aggregation_level``` (https://github.com/lwyszomi/commcare-hq/blob/13d993b82de9c42e7b6b86f48b62928d3452ecd5/custom/icds_reports/migrations/sql_templates/create_datasource_views.sql#L19) which is created in migration ```0016_aggawcmonthly_aggccsrecordmonthly_aggchildhealthmonthly_aggdailyusageview_aggthrmonthly_awclocation_```
and ```num_launched_states``` (https://github.com/lwyszomi/commcare-hq/blob/13d993b82de9c42e7b6b86f48b62928d3452ecd5/custom/icds_reports/migrations/sql_templates/create_datasource_views.sql#L440) which is created ```0017_add_v2_indicators```

Without this fix it raises error that those column don't exist.